Source/levels/themes.cpp

```diff
@@ -854,6 +854,8 @@ void InitThemes()
 				themes[numthemes].ttype = j;
 				numthemes++;
 			}
+			if (i == std::numeric_limits<int8_t>::max())

```

Comment:
Previously the maximum was 255, now it's 127, is that an issue?

Reply:
`MAXTHEMES 50` so should be good

Reply:
Actually not so sure :D

This was previously looking in 255 places for where to put 50 rooms. But on the bright side the largest number I have seen so fare is 49 possible places, we are currently scanning to see if we can find some levels with more then that :)